lib/AST/ASTPrinter.cpp

```diff
@@ -4037,7 +4037,9 @@ class TypePrinter : public TypeVisitor<TypePrinter> {
     visit(staticSelfT);
   }
 
-  void printFunctionExtInfo(ASTContext &Ctx, AnyFunctionType::ExtInfo info) {
+  void printFunctionExtInfo(AnyFunctionType *fnType) {
+    auto &Ctx = fnType->getASTContext();
```

Comment:
nit: `ctx` not `Ctx`.

That may be out of the scope of this PR, though.

Reply:
Fixed.